pkgs/development/python-modules/pydata-sphinx-theme/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, sphinx
+, beautifulsoup4
+, docutils
+, packaging
+}:
+
+buildPythonPackage rec {
+  pname = "pydata-sphinx-theme";
+  version = "0.8.1";
+
+  format = "wheel";
```

Comment:
Why are we fetching a wheel here?

Reply:
because building it from source it a bit complicated, it needs `webpack` and other node packages

Reply:
yeah, webpack things are a bit complicated but this makes patching a bit harder.